addons/sourcemod/scripting/shavit-core.sp

```diff
@@ -226,26 +249,18 @@ public void OnPluginStart()
 	{
 		SetFailState("This plugin was meant to be used in CS:S, CS:GO and TF2 *only*.");
```

Comment:
This is completely wrong and breaks the whole plugin for CS:S and TF2. Also, this function should also handle `sv_autobunnyhopping` as it's CS:GO specific.